vnext/Scripts/Microsoft.ReactNative.Managed.nuspec

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <description>Contains Windows Implementation of React-Native for Managed code</description>
+    <authors>Microsoft</authors>
+    <projectUrl>https://github.com/microsoft/react-native-windows</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <repository type="git"
+      url="https://github.com/microsoft/react-native-windows.git"
+      commit="$CommitId$" />
+    <tags>react react-native native-module microsoft c# csharp</tags>
+    <dependencies>
+      <group targetFramework="UAP10.0">
+        <dependency id="Microsoft.UI.Xaml" version="2.3.191129002" />
+        <dependency id="Microsoft.NETCore.UniversalWindowsPlatform" version="6.2.9" />
+        <dependency id="Microsoft.ReactNative" version="$version$" />
+      </group>
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="$nugetroot$\$baseplatform$\$baseconfiguration$\Microsoft.ReactNative.Managed\Microsoft.ReactNative.Managed\ref\Microsoft.ReactNative.Managed.dll"   target="ref\uap10.0"/>
+    <!-- 
+    Uncomment once we are producing doc xml file
```

Comment:
>Uncomment [](start = 4, length = 9)

It seems the project already set to produce this file. Should it be uncommented?

Reply:
yes :)